collectors/mmrepquota_test.go

```diff
@@ -35,6 +35,20 @@ mmrepquota::0:1:::project:FILESET:408:PZS1003:341467872:2147483648:2147483648:0:
 *** Report for FILESET quotas on scratch
 mmrepquota::HEADER:version:reserved:reserved:filesystemName:quotaType:id:name:blockUsage:blockQuota:blockLimit:blockInDoubt:blockGrace:filesUsage:filesQuota:filesLimit:filesInDoubt:filesGrace:remarks:quota:defQuota:fid:filesetname:
 mmrepquota::0:1:::scratch:FILESET:0:root:928235294208:0:0:5308909920:none:141909093:0:0:140497:none:i:on:off:::
+*** Report for USR quotas on home
```

Comment:
For testing, I think it'd be good to leave `mmrepquotaStdout` as it was before and add like `mmrepquotaStdoutAll` that is that original stdout plus the user/group logic and test that after testing the old default behavior.  Just extra test to ensure that the parsing of default fileset data still works like before.

Reply:
Fixed. I've created separate tests with and without the users and groups.